sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/AppendBlobAsyncClient.java

```diff
@@ -86,16 +86,20 @@
      *         {@link BlobAccessConditions}
      *
      * @return
-     *      A reactive response containing the information of the created appended blob.
+     *      A {@link Mono} containing {@link Response} whose {@link Response#value() value} contains the {created appended blob.
```

Comment:
Random `{` that'll break JavaDoc generation

Reply:
yup, removed!